frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
+  SystemStyleObjectRecord,
+  ThemingPropsThunk,
+} from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+import merge from 'lodash/merge'
+
+export const DRAG_SCROLL_SPEED = 0.8
+
+const variantLine: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> =
+  () => {
+    return {
+      root: {
+        w: '100%',
+      },
+      tablist: {
+        w: '100%',
```

Comment:
May need `white-space: nowrap` properties too (following instructions from the use-drag-scroll package)